test/parallel/test-tls-getprotocol.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+if (!common.hasCrypto) {
+  console.log('1..0 # Skipped: missing crypto');
+  return;
+}
+
+const tls = require('tls');
+const fs = require('fs');
+
+const options = {
+  key: fs.readFileSync(common.fixturesDir + '/keys/agent2-key.pem'),
+  cert: fs.readFileSync(common.fixturesDir + '/keys/agent2-cert.pem'),
+  secureProtocol: 'TLSv1_2_method'
+};
+
+tls.createServer(options, common.mustCall(function() {
+  this.close();
+})).listen(common.PORT, '127.0.0.1', function() {
+  tls.connect({
+    host: '127.0.0.1',
+    port: common.PORT,
+    rejectUnauthorized: false
+  }, function() {
+    assert.strictEqual(this.getProtocol(), 'TLSv1.2');
+    this.end();
```

Comment:
No, `getProtocol()` will return `null` for disconnected client sockets, just like the other `get*()` methods on TLS sockets.